src/sentry/tasks/statistical_detectors.py

```diff
@@ -241,6 +303,31 @@ def _detect_transaction_trends(
     )
 
 
+def query_transactions_timeseries(
+    transactions: List[Tuple[int, int | str]],
+    start: datetime,
+    agg_function: str,
+) -> Generator[Tuple[int, int, Any], None, None]:
+    end = start.replace(minute=0, second=0, microsecond=0) + timedelta(hours=1)
+    interval = 3600  # 1 hour
+
+    for project_id, transaction_name in transactions:
+        params: Dict[str, Any] = {
+            "start": end - timedelta(days=14),
+            "end": end,
+            "project_id": [project_id],
+            "project_objects": [Project.objects.get(id=project_id)],
+        }
+        results = metrics_performance.timeseries_query(
+            selected_columns=[agg_function],
+            query=f"transaction:#{transaction_name}",
```

Comment:
not sure `#` is needed here since we don't do any special handling for it so it might just be interpreted as part of the transaction name

Reply:
Good catch! Sorry, Ruby muscle memory